examples/qinstall-config.yml

```diff
@@ -0,0 +1,264 @@
+jumpbox:
+  vm_size: Standard_B2ms
+  # SSH port under which the jumpbox SSH server listens on the public IP. Default to 22
+  # Change this to, e.g., 2222, if security policies (like "zero trust") in your tenant automatically block access to port 22 from the internet
+  #ssh_port: 2222
+# Active directory VM configuration
+ad:
+  vm_size: Standard_B2ms
+  hybrid_benefit: false # Enable hybrid benefit for AD, default to false
+  high_availability: false # Build AD in High Availability mode (2 Domain Controlers) - default to false
+# On demand VM configuration
+ondemand:
+  vm_size: Standard_D4s_v5
+  #fqdn: azhop.foo.com # When provided it will be used for the certificate server name
+  generate_certificate: true # Generate an SSL certificate for the OnDemand portal. Default to true
+# Grafana VM configuration
+grafana:
+  vm_size: Standard_B2ms
+# Guacamole VM configuration
+guacamole:
+  vm_size: Standard_B2ms
+# Scheduler VM configuration
+scheduler:
+  vm_size: Standard_B2ms
+# CycleCloud VM configuration
+cyclecloud:
+  vm_size: Standard_B2ms
+  # version: 8.3.0-3062 # to specify a specific version, see https://packages.microsoft.com/yumrepos/cyclecloud/
+
+# List of users to be created on this environment
+users:
+  - { name: clusteradmin, uid: 10001, groups: [5001, 5002] }
+  - { name: hpcuser, uid: 10002 }
+
+usergroups:
+# These groups can’t be changed
+  - name: Domain Users # All users will be added to this one by default
+    gid: 5000
+  - name: az-hop-admins
+    gid: 5001
+    description: "For users with azhop admin privileges"
+  - name: az-hop-localadmins
+    gid: 5002
+    description: "For users with sudo right or local admin right on nodes"
+
+# scheduler to be installed and configured (openpbs, slurm)
+queue_manager: openpbs
+
+# Specific SLURM configuration
+slurm:
+  # Enable SLURM accounting, this will create a SLURM accounting database in a managed MariaDB server instance
+  accounting_enabled: false
+  # SLURM version to install. Currently supported: only 20.11.9 and 22.05.3.
+  # Other versions can be installed by building from source (See build_rpms setting in the slurmserver role)
+  slurm_version: 20.11.9
+  # Name of the SLURM cluster for accounting (optional, default to 'slurm')
+  # WARNING: changing this value on a running cluster will cause slurmctld to fail to start. This is a
+  # safety check to prevent accounting errors. To override, remove /var/spool/slurmd/clustername
+  cluster_name: slurm_azhop
+
+enroot:
+  enroot_version: 3.4.1
+
+# Authentication configuration for accessing the az-hop portal
+# Default is basic authentication. For oidc authentication you have to specify the following values
+# The OIDCClient secret need to be stored as a secret named <oidc-client-id>-password in the keyvault used by az-hop
+authentication:
+  httpd_auth: basic # oidc or basic
+
+image_gallery:
+  create: true # Create the shared image gallery to store custom images
```

Comment:
We don't need an image gallery as we are using marketplace images. You should be able to remove the whole section

Reply:
Removing image gallery.